src/site/content/en/lighthouse-pwa/themed-omnibox/index.md

```diff
@@ -1,70 +1,79 @@
 </figure>
 
-## How this audit fails
-
 The audit fails if Lighthouse doesn't find a `theme-color` meta tag in the page's
-HTML and a `theme_color` property in the Web App Manifest.
-Lighthouse does not test whether the values are valid CSS color values.
+HTML and a `theme_color` property in the [web app manifest](/add-manifest).
 
-{% include 'content/lighthouse-pwa/scoring.njk' %}
+Note that Lighthouse doesn't test whether the values are valid CSS color values.
 
-## Recommendations
+{% include 'content/lighthouse-pwa/scoring.njk' %}
 
-To ensure that the address bar is always themed to your colors:
+## How to set a theme color for the address bar
 
-1. Add a `theme-color` meta tag to the HTML of every page you want to brand.
-2. Add the `theme_color` property to your Web App Manifest.
+### Step 1: Add a `theme-color` meta tag to every page you want to brand
 
 The `theme-color` meta tag ensures that the address bar is branded when
-a user visits your site as a normal webpage. Set `content` to any valid CSS
-color value. You need to add this meta tag to every page that you want to
-brand.
+a user visits your site as a normal webpage.
+Set the tag's `content` attribute to any valid CSS color value:
 
-```html
+```html/4
+<!DOCTYPE html>
+<html lang="en">
 <head>
+  …
   <meta name="theme-color" content="#317EFB"/>
-  ...
+  …
+</head>
+…
 ```
 
-Learn more about `theme-color` in
-[Support for theme-color in Chrome 39 for Android](https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android).
+Learn more about the `theme-color` meta tag in Google's
+[Support for `theme-color` in Chrome 39 for Android](https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android).
+
+### Step 2: Add the `theme_color` property to your web app manifest
 
-The `theme_color` property in your Web App Manifest ensures that the address
-bar is branded when a user launches your progressive web
-app from the homescreen. Unlike the `theme-color` meta tag, you only need
-to define this once, in the manifest. The browser colors every page of your
-app according to the manifest's `theme_color`. Set the property to any valid
-CSS color value.
+The `theme_color` property in your web app manifest ensures that the address
+bar is branded when a user launches your PWA from the home screen.
+Unlike the `theme-color` meta tag, you only need
+to define this once, in the [manifest](/add-manifest).
+Set the property to any valid CSS color value:
 
-```html
+```html/1
 {
   "theme_color": "#317EFB"
-  ...
+  …
 }
  ```
 
-See [User can be prompted to install the web app](/installable-manifest)
-for more resources on adding a manifest to your app.
+<!-- TODO
+Confirm the following sentence with Pete/Joe. It makes it sound like
+theme_color overrides the page's background-color.
+-->
+ The browser will color every page of your app
```

Comment:
```suggestion
 The browser will color the address bar of every page of your app
```

I think this fixes the TODO above

Reply:
Thanks. Didn't want to change the meaning without being sure of behavior.